api/controller/auth.js

```diff
@@ -19,6 +20,21 @@ const generateToken = (user) => {
   }
 };
 
+// Access Token 1hr
+const generateAccessToken = (access) => {
+  const payload = {
+    exp:
+      Math.floor(Date.now() / 1000) + parseInt(process.env.ACCESS_TOKEN_EXPIRY),
+    data: access,
+  };
+  try {
+    return jwt.sign(payload, secret);
+  } catch (error) {
+    console.error("[Error] Failed to generate Access token:", error);
+    throw error;
+  }
+};
+
```

Comment:
Also can you please explain about this function? Difference between existing `generateToken` ??

Reply:
nothing to be on big it will be add a time of different like 1hour, 2hour different times,  now i will update code  parameter now we can get a 2 parameter first one for data, 2nd one for time.